### PR TITLE
Migration: Updates contrib values migration

### PR DESCRIPTION
**Pull request by dgasmith**
## Status
- [ ] Changelog updated
- [x] Ready to go

### CODE REVIEW

File: qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py [hunk 1]
```diff
@@ -22,10 +22,6 @@ def upgrade():
     op.add_column('collection', sa.Column('group', sa.String(), nullable=False, server_default='default'))
     op.alter_column('collection', 'group', server_default=None)
 
-    op.add_column('collection', sa.Column('metadata', sa.JSON(), nullable=True))
-    op.execute("UPDATE collection SET metadata = '{}'::json")
-    op.alter_column('collection', 'metadata', nullable=False)
-
```
Comment (doaa-altarawy) [inline qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py]: This is on purpose, right?

Comment (dgasmith) [reply qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py]: Yes, @mattwelborn SQLAlchemy does not allow you to have a `metadata` field.d

Comment (mattwelborn) [reply qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py]: do you mean that "metadata" is a reserved name?

Comment (dgasmith) [reply qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py]: A reserved class attribute, yes.

**Pull request merged**